src/interfaces/IPlugin.sol

```diff
@@ -1,7 +1,10 @@
+// solhint-disable one-contract-per-file
```

Comment:
Slight preference for defining each of these interfaces in its own file.

Reply:
Done!